.github/workflows/build-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       YARN_VERSION: ${{ steps.yarn-version.outputs.YARN_VERSION }}
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [14.x, 16.x, 18.x, 19.x]
```

Comment:
Hmm, we haven't typically tested non-LTS versions.  I guess because they sometimes have stability issues that we don't care about, because we aren't eager enough to try using them.

But if that happens in a particular package we can just stop testing it. This seems good and useful.

Reply:
after having had this mulled for a bit: IMO it would make sense to have different default policies for library packages (down to oldest/latest maintenance) vs applications/services (lts enough). wdyt?

Reply:
That seems reasonable!

This template is intended for libraries in particular. We have few enough applications/services that we haven't created a template for them.

Reply:
makes sense!